deployment/manifests/demo/demo_deployment.yaml

```diff
@@ -0,0 +1,52 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: demo-exome-results-browsers
+  labels:
+    service: demo-exome-results-browsers
+    environment: qa
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: demo-exome-results-browsers
+  template:
+    metadata:
+      labels:
+        name: demo-exome-results-browsers
+    spec:
+      containers:
+        - name: app
+          image: gcr.io/exac-gnomad/exome-results-browsers
+          imagePullPulicy: Always
+          env:
+            - name: RESULTS_DATA_DIRECTORY
+              value: /mnt/disks/erb-data/results
+            - name: TRUST_PROXY
+              value: '2'
+          ports:
+            - name: http
+              containerPort: 8000
+          resources:
+            requests:
+              cpu: 100m
+              memory: 256Mi
+            limits:
+              cpu: 100m
+              memory: 512Mi
+          readinessProbe:
+            httpGet:
+              path: /ready
+              port: http
+            initialDelaySeconds: 5
+            periodSeconds: 30
+          volumeMounts:
+            - name: data
+              mountPath: /mnt/disks/erb-data
+      volumes:
+        - name: data
+          gcePersistentDisk:
+            fsType: ext4
+            pdName: exome-results-browsers-data-230123-1052-east1c
```

Comment:
This refers to a specific disk in GCE: https://console.cloud.google.com/compute/disksDetail/zones/us-east1-c/disks/exome-results-browsers-data-230123-1052-east1c?project=exac-gnomad

When you deploy this browser to production, you'll likely populate the data on that disk, but for development, you may wish to update the `pdName` to point at a new disk. Since this is using a `gcePersistentDisk` volume, you need to create it by hand (i.e. it's not automatically allocated like disks in some of our other deployments are).